flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -377,6 +378,23 @@ void createIcebergTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
     PartitionSpec spec = toPartitionSpec(((CatalogTable) table).getPartitionKeys(), icebergSchema);
 
     ImmutableMap.Builder<String, String> properties = ImmutableMap.builder();
+
+    List<WatermarkSpec> watermarkSpecs = table.getSchema().getWatermarkSpecs();
```

Comment:
It seems that `TableSchema` is deprecated. Should we switch to `getUnresolvedSchema` here?

Reply:
Yes, it should be update. But there is many other work relate to this, so maybe should other PR to update this.